examples/solverdummies/precice-config.xml

```diff
@@ -51,8 +51,8 @@
     <participants first="SolverOne" second="SolverTwo" />
     <max-time-windows value="2" />
     <time-window-size value="1.0" />
+    <min-iterations value="2" />
     <max-iterations value="2" />
```

Comment:
Open for discussion, see #800

Reply:
After the conclusion in #800, the `min-itertations` should go.